examples/cfd/swe_distributed_gnn/shallow_water_equations.py

```diff
@@ -0,0 +1,463 @@
+# coding=utf-8
```

Comment:
Need to include # ignore_header_test to pass the license test

Reply:
done